src/claim/Claim.utils.ts

```diff
@@ -33,7 +33,7 @@ export function compress(claim: IClaim): CompressedClaim {
  *  Decompresses the [[Claim]] from storage and/or message.
  *
  * @param claim A compressesd [[Claim]] array that is reverted back into an object.
- *
+ * @throws When [[claim]] is not an Array or it's length is unequal 3.
```

Comment:
does this work, when using lowercase for `[[claim]]`?

Reply:
Capitalized